### ISSUE TITLE
Style/VariableNumber with snake_case doesn't allow numbers >9

### ISSUE THREAD

**Issue by savef** [bbatsov/rubocop#3519]
There are 2 other issues (#3514 and #3516) related to this new cop but I don't think either covers this case. To be brief, when configured with `EnforcedStyle: snake_case` and using the following code:
```ruby
  get_0 = timer.get(0)
  get_10 = timer.get(10)
  get_25 = timer.get(25)
```
I see offences for `get_10` and `get_25`, but not `get_0`. I think this is simply a regex issue where more than 1 numeric character at the end is causing an offence when it should not.

-----
```
$ rubocop -V
0.43.0 (using Parser 2.3.1.3, running on ruby 2.3.1 x86_64-linux)
```

**Comment by bbatsov** [comment]
I think this is fixed on master now.
